nautobot/users/factory.py

```diff
@@ -1,21 +1,36 @@
 from datetime import timezone
 
 from django.contrib.auth import get_user_model
+from django.urls import NoReverseMatch, reverse
 import factory
 
-from nautobot.core.factory import BaseModelFactory, NautobotBoolIterator
+from nautobot.core.factory import BaseModelFactory, NautobotBoolIterator, random_instance, UniqueFaker
+from nautobot.extras.utils import FeatureQuery
+from nautobot.users.models import SavedView
 
 User = get_user_model()
 
+VIEW_NAMES = []
+for choice in FeatureQuery("saved_views").get_choices():
```

Comment:
Looks like this is failing in CI because it's hitting the DB at import time before the DB is ready. This probably needs to be moved into a function of some sort.